### DISCUSSION TITLE
When building on Linux, it does not generate the libduckdb_java.so_windows_amd64 file.

### DISCUSSION THREAD

**Discussion by yundg** [duckdb/duckdb#6955] [Q&A]
I built it through the CLI below.
`BUILD_JDBC=1 make`
Only the libduckdb_java.so_linux_amd64 file exists in the jdbc jar file.

Is it possible to build to include the files below?
- libduckdb_java.so_linux_arm64 
- libduckdb_java.so_windows_amd64
- libduckdb_java.so_osx_universal

**Comment by Mause** [comment]
It's possible to build the other files on their native systems. To build for windows for example, you must build on a windows machine
